app/controllers/api/v1x0.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V1x0
     class RootController < ApplicationController
       def openapi
-        render :json => Rails.root.join('public', 'catalog', 'v1.0.0', 'openapi.json').read
+        render :json => Rails.root.join('public', 'doc', 'openapi-3-v1.0.0.json').read
```

Comment:
Should we match syntax with Topo?  

https://github.com/ManageIQ/topological_inventory-api/blob/master/app/controllers/api/v1x0.rb#L4

`render :json => ::ManageIQ::API::Common::OpenApi::Docs.instance["1.0"].to_json`